src/styles/globals.css

```
@@ -8,6 +8,8 @@
 @import "./shiki.css";
 @import "./video.css";
 
+@import "./slider.css";
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)